src/javascript/app/base/footer.js

```diff
@@ -15,8 +15,11 @@ const Footer = (() => {
             if (Client.isLoggedIn()) {
                 showWarning((Client.get('landing_company_shortcode') === 'maltainvest' ||
                     (Client.get('is_virtual') && State.getResponse('landing_company.financial_company.shortcode') === 'maltainvest')));
+                showAgeRestrictionSign(/iom|maltainvest/.test(Client.get('landing_company_shortcode')) ||
+                    (Client.get('is_virtual') && /iom|maltainvest/.test(Client.get('landing_company_shortcode'))));
             } else {
                 showWarning(isEuCountry());
+                showAgeRestrictionSign(isEuCountry());
```

Comment:
we are duplicating `isEuCountry()`, better store it in a variable inside this `else`